unime/src/i18n/en/index.ts

```diff
@@ -231,7 +264,11 @@ const en = {
       DESCRIPTION: 'Try searching for something else.',
     },
   },
+  CREDENTIAL: {
+    NAVBAR_TITLE: 'Credential info',
+  },
   BADGE: {
+    NAVBAR_TITLE: 'Badge info',
```

Comment:
It used to be "Badge information". Now it's "Badge info". Was that by accident or on purpose?